packages/core/src/card/card.stories.ts

```diff
@@ -0,0 +1,292 @@
+    Now we routinely download whole albums worth of music in a couple of minutes.`;
+const longerContent = `
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    Praesent volutpat tortor eget quam auctor, quis sagittis libero auctor.
+    Nulla augue ante, tincidunt sit amet semper vitae, tempus at ipsum.
+    Vestibulum elementum, turpis quis ullamcorper fermentum, elit turpis placerat ipsum, quis convallis ex nisi sit amet lacus.
+    Ut enim ipsum, tincidunt nec luctus id, pharetra id velit. Aliquam nec elit ut neque lacinia mattis id ac lorem.
+    Vivamus egestas massa nulla, ac elementum purus pretium eu. Duis ultrices nec tortor varius consectetur.
+    Maecenas convallis bibendum urna, et aliquam quam tincidunt in.
+    Nam viverra, leo vitae imperdiet ultrices, ante mi facilisis mi, ac vehicula erat nulla sed nibh.
+    Nam euismod gravida est, convallis placerat nunc pharetra sed. Nullam scelerisque dui augue, eu porta erat tempus in.
+
+    Integer egestas massa orci, id scelerisque libero lobortis vitae.
+    Vivamus aliquet sem massa, eget sodales sapien dictum eu.
+    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Suspendisse non interdum sem, nec egestas nulla.
+    Duis vitae leo justo. Fusce ante enim, tincidunt nec faucibus sed, fringilla et ipsum.
+    Proin quis diam elit. Cras nec finibus libero. Sed bibendum lectus eget magna ultricies eleifend.
+    Duis ut molestie urna. Proin cursus dolor ac ex rutrum, eu condimentum elit tincidunt.
+    In a dolor faucibus, vehicula ipsum eu, viverra lorem. Aenean congue fermentum commodo.
+    Sed ex purus, consectetur mollis urna eget, vestibulum mollis dolor.
+    Donec sodales, felis sit amet gravida ornare, magna dolor ullamcorper turpis, vel facilisis odio ipsum semper sem.
+  `;
+
+export default {
+  title: 'Stories/Card',
+  component: 'cds-card',
+  argTypes: getElementStorybookArgTypes('cds-card', customElements),
+  parameters: {
+    options: { showPanel: true },
+  },
+};
+
+export function Basic(args: { [key: string]: unknown }) {
+  return html`<cds-card ...="${spreadProps(getElementStorybookArgs(args))}">
+    <p cds-text="body">${placeholder}</p>
+  </cds-card>`;
+}
+
+export function WithLayout() {
+  return html` <cds-card>
+    <div cds-layout="vertical gap:md">
+      <div cds-text="section" cds-layout="p-y:sm">
+        Card Title
+      </div>
+
+      <div cds-text="body" cds-layout="p-y:lg">
+        ${basicContent}
+      </div>
+
+      <cds-divider cds-card-remove-margin></cds-divider>
+
+      <div cds-layout="horizontal gap:sm p-y:sm align:vertical-center"></div>
+    </div>
+  </cds-card>`;
+}
+
+export function WithContainerOfCards() {
+  return html`
+    <style>
+      cds-card.cds-card__fit {
+        flex: 1;
+      }
+    </style>
+    <div cds-layout="horizontal gap:md">
```

Comment:
Non blocker: if you change this layout to use grid then the flex style is not needed and it will be responsive.

```html
<div cds-layout="grid cols@md:6 cols@lg:3 gap:md">
```